certbot-nginx/local-oldest-requirements.txt

```diff
@@ -1,3 +1,3 @@
 # Remember to update setup.py to match the package versions below.
-acme[dev]==0.29.0
-certbot[dev]==0.36.0
+-e acme[dev]
+-e .[dev]
```

Comment:
I think we should update `certbot-nginx/setup.py` to reflect these changes as well since we're no longer testing with older versions.